rules/java-jdbc/src/test/java/io/opentracing/contrib/agent/jdbc/JdbcITest.java

```diff
@@ -0,0 +1,39 @@
+package io.opentracing.contrib.agent.jdbc;
+
+/**
+ * Created by jam01 on 9/13/18.
+ */
+
+import io.opentracing.Scope;
+import io.opentracing.contrib.agent.common.OTAgentTestBase;
+import io.opentracing.mock.MockSpan;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author malafeev
+ * @author Jose Montoya
+ */
+public class JdbcITest extends OTAgentTestBase {
+
+	@Test
+	public void test() throws Exception {
+		try (Scope ignored = getTracer().buildSpan("jdbc-test").startActive(true)) {
+
+			Connection connection = DriverManager.getConnection("jdbc:h2:mem:jdbc");
+			Statement statement = connection.createStatement();
+			statement.executeUpdate("CREATE TABLE employer (id INTEGER)");
+			connection.close();
+		}
+
+		List<MockSpan> spans = getTracer().finishedSpans();
```

Comment:
Could you add a comment regarding why 2 spans - just to make it clear one is for the parent span explicitly created, and the other for the db update. Alternatively could move the check into the try block after the `executeUpdate` and only check for the single span?